src/universal/modules/summary/containers/MeetingSummary/MeetingSummaryContainer.js

```diff
@@ -71,6 +73,7 @@ const mapStateToProps = (state, props) => {
 
 @requireAuth
 @connect(mapStateToProps)
+@maintainSocket
```

Comment:
cancels a destruction if one exists. 
when the component unmounts, it checks to see if a socket is used anywhere else. if this is the last hoc standing, then it starts a new countdown.